parser.go

```diff
@@ -104,6 +104,7 @@ func moveToRoom(exit string) {
 	if eagleWatching {
 		fmt.Printf("AGH! The eagle is taking his revenge!\n")
 		tauntTheEagle()
+		eagleWatching = false
```

Comment:
Do we need this? The first thing tauntTheEagle() does is set eagleWatching = false.  

The only place eagleWatching is set true is lookAtEagle, which is only called if the player is in the Yard. 
From there you can useTheUmbrella or you can tauntEagle. If you try to keep going and ignore the eagle, tauntEagle is called for you.